src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6981,7 +6986,9 @@ namespace OutputReportTabular {
 
 			// get change in overall state of charge for electrical storage devices.
 			if ( NumElecStorageDevices > 0 ) {
-				OverallNetEnergyFromStorage = ( sum( ElecStorage.StartingEnergyStored() ) - sum( ElecStorage.ThisTimeStepStateOfCharge() ) );
```

Comment:
A subtle change and an even subtler build warning.  I'll suggest a change to CI that helps provide more context from the compiler output.